docs/src/pages/docs/functions/logic-functions.md

```diff
@@ -263,6 +263,68 @@ back a predicate function ready for insertion into your flow. All predicate
 based functions in `crocks` take either a [`Pred`][pred] or predicate
 function, so it should be easy to swap between the two.
 
+```javascript
+import not from 'crocks/logic/not'
+
+import Pred from 'crocks/Pred'
+import propSatisfies from 'crocks/predicates/propSatisfies'
+import isString from 'crocks/predicates/isString'
+import and from 'crocks/logic/and'
+import compose from 'crocks/core/compose'
+import flip from 'crocks/combinators/flip'
+import identity from 'crocks/combinators/identity'
+
+// isFalsy :: a -> Boolean
+const isFalsy = not(identity)
+
+// isTruthy :: a -> Boolean
+const isTruthy = not(isFalsy)
+
+isTruthy('Test string')
+//=> true
+
+isTruthy('')
+//=> false
+
+isFalsy('')
+//=> true
+
+isFalsy('Test string')
+//=> false
+
+// User :: { String, String, String}
+
+const validUser = {
+  email: 'testuser@email.com',
+  firstName: 'Tom',
+  lastName: 'Smith'
+}
+
+const invalidUser = {
+  email: '',
+  firstName: '',
+  lastName: 'Smith'
+}
+
+// hasValidStringProp :: string -> User -> Boolean
+const hasValidStringProp = compose(Pred, flip(propSatisfies, and(isString, isTruthy)))
```

Comment:
Spliting this up into simple part and using multi-line definitions can make it easier to follow:
```js
// isNonEmptyString :: a -> Boolean
const isNonEmptyString =
  and(isString, isTruthy)

// hasValidStringProp :: String -> User -> Boolean
const hasValidStringProp = compose(
  Pred,
  flip(propSatisfies, isNonEmptyString)
)
```